qcfractal/queue/base_adapter.py

```diff
@@ -13,20 +13,33 @@ class BaseAdapter(abc.ABC):
     """A BaseAdapter for wrapping compute engines
     """
 
-    def __init__(self, client: Any, logger: Optional[logging.Logger] = None):
+    def __init__(self,
+                 client: Any,
+                 logger: Optional[logging.Logger] = None,
+                 **kwargs):
```

Comment:
Can we have `cores_per_task` and memory here explicitly rather than relying on kwargs for just tow?

Reply:
Reminder here.